api/Worklet.json

```diff
@@ -66,6 +66,45 @@
             "deprecated": false
           }
         }
+      },
+      "ecmascript_modules": {
```

Comment:
The spec states that only static `import` is supported (not `import()`. Should we:
- Make it something like `import_supported` with description "Supports static import of ecmascript modules (only, as per spec)"

Reply:
No, what I have done is good. Docs include info about what aspects of ecmascript module support are covered by spec.